app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/detailLibraryActivity.java

```diff
@@ -0,0 +1,86 @@
+package com.cmput301f20t21.bookfriends.ui.library;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.cmput301f20t21.bookfriends.R;
+import com.cmput301f20t21.bookfriends.entities.Book;
+import com.cmput301f20t21.bookfriends.enums.BOOK_ACTION;
+import com.cmput301f20t21.bookfriends.repositories.BookRepository;
+import com.cmput301f20t21.bookfriends.ui.add.AddEditActivity;
+import com.cmput301f20t21.bookfriends.utils.GlideApp;
+import com.google.firebase.storage.FirebaseStorage;
+import com.google.firebase.storage.StorageReference;
+
+public class detailLibraryActivity extends AppCompatActivity {
+    public static final String BOOK_ACTION_KEY = "com.cmput301f20t21.bookfriends.BOOK_ACTION";
+    public static final String BOOK_EDIT_KEY = "com.cmput301f20t21.bookfriends.BOOK_EDIT";
+    private Book detailBook;
+    private ImageView bookImage;
+    private TextView detailTitle;
+    private TextView detailISBN;
+    private TextView detailAuthor;
+    private final BookRepository bookRepository = BookRepository.getInstance();
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.detail_borrow);
```

Comment:
question: why is the layout name detail_borrow? doesn't seem to match the content